src/install/browserPaths.ts

```diff
@@ -77,9 +77,30 @@ export function executablePath(browserPath: string, browser: BrowserDescriptor):
   return tokens ? path.join(browserPath, ...tokens) : undefined;
 }
 
+const defaultBrowsersPath = ((): string | undefined => {
+  const env = getFromENV('PLAYWRIGHT_BROWSERS_PATH');
+  if (env === '0')
+    return undefined;
+
+  let result: string;
+  if (env)
+    result = env;
+  else if (process.platform === 'linux')
+    result = '~/.cache/playwright';
```

Comment:
Let's do `playwright-nodejs` suffix to isolate from conflicts with native `playwright` application.

Reply:
Done